pkg/build/interfaces.go

```diff
@@ -49,6 +49,12 @@ type Ignorer interface {
 	Ignore(*api.Config) error
 }
 
+// Runner allows the user to actually run the produced imgae as part of the s2i
+// cmd line invocation, to see if it operates as he would expect
+type Runner interface {
+	Run(*api.Config) error
+}
```

Comment:
There's a bigger plan to abstract the entire flow of s2i build to remove unnecessary code duplication we have in origin. Let's first do that and then think about adding additional interfaces if you don't mind. Besides OS will never leverage that Runner interface in the first place, it's solely for s2i standalone edition.